central/reports/scheduler/schedule.go

```diff
@@ -123,6 +128,28 @@ type reportEmailFormat struct {
 	DateStr    string
 }
 
+// Adds RHACS or Stackrox branding to email vulnerability report based on image flavor
+func getBrandedNoVulnsFoundEmailTemplate() string {
+	var brandedNoVulnsFoundEmailTemplate string
+	if defaults.ImageFlavorEnv() == defaults.ImageFlavorNameStackRoxIORelease {
```

Comment:
This is incorrect. Unfortunately, our naming isn't great, but the `stackrox.io` flavor _isn't_ an indicator that we're on the OpenSource branding. This flavor is used by the commercial customers that are still on the legacy `stackrox.io` image repository. If we implement it like this, commercial RHACS customers getting images from `stackrox.io` will effectively see StackRox branding on vuln templates. 

I see two ways forward:
1) We continue to rely on image flavor, @Maddosaurus is implementing an `opensource` flavor, and then we do the check on that one instead.
2) We add a new environment variable for managing the BE branding. Similar to the `REACT_APP_ROX_PRODUCT_BRANDING` that will be set on `.circleci` and downstream images. 

The problem with (1) is that if we tie image flavor to branding, what do we do with `development_build` flavor? Both the community and internal developers are building with that flavor. For simplicity, we could always assume that development = open source branding. But that's not an optimal solution. 

I'm more inclined towards (2). @msugakov do you have any preference?

Reply:
Another small point in favor of (2) is that we won't need to make the environment variable `ImageFlavorEnv()` public in `pkg/images/defaults`.

Reply:
I asked the same question to Misha too, but what exactly is the point of adding a new environment variable instead of reusing the already existing `REACT_APP_ROX_PRODUCT_BRANDING` when they would have to match at all times anyways?

Reply:
`REACT_APP_ROX_PRODUCT_BRANDING` is a build time environment that is currently only used to build the FE app.

Reply:
I foresee another problem with (1): If we tie (& default) branding and flavor like this, we're adding more and more work for the FE folks to override this if they develop & test in both UI brandings.